x-pack/plugins/security_solution/cypress/integration/detection_rules/indicator_match_rule.spec.ts

```diff
@@ -168,6 +169,11 @@ describe('indicator match', () => {
           getDefineContinueButton().click();
           getIndexPatternInvalidationText().should('exist');
         });
+
+        it('Shows invalidation text if you try to create invalid index pattern', () => {
```

Comment:
A new end-to-end test feels very heavy-handed here. We could perhaps add this assertion to an existing "rule form validations" test if we had one, but the overhead is not worth the test coverage, here, IMO.

The validator is a pure function that should be straightforward to unit test.